sentry_sdk/tracing.py

```diff
@@ -303,7 +391,8 @@ def record_http_request(hub, url, method):
                     span.set_data(k, v)
 
 
-def maybe_create_breadcrumbs_from_span(hub, span):
+def _maybe_create_breadcrumbs_from_span(hub, span):
+    # type: (sentry_sdk.Hub, Span) -> None
     if span.op == "redis":
         hub.add_breadcrumb(type="redis", category="redis", data=span._tags)
     elif span.op == "http" and span.is_success():
```

Comment:
You don't want failed http requests to be breadcrumbed as well?

Reply:
The old Python SDK didn't do that, and I don't want to change this UI for now.